src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java

```diff
@@ -819,7 +820,7 @@ public void onEntityExplodeChangeBlock(final EntityExplodeEvent event) {
 
             // don't bother record upper doors.
             if (MaterialTag.DOORS.isTagged(block.getType())) {
-                if (((Door) block.getState().getData()).isTopHalf()) {
+                if (((Door) block.getState().getBlockData()).getHalf() == Bisected.Half.TOP) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5490548)